plugins/modules/ec2_eni.py

```diff
@@ -410,6 +404,11 @@ def wait_for(function_pointer, *args):
             break
 
 
+def _generate_tag_specifications(tags):
+    tag_list = ansible_dict_to_boto3_tag_list(tags)
+    return list(dict(ResourceType="network-interface", Tags=tag_list))
```

Comment:
This is more efficient:
```suggestion
    return ["ResourceType", "Tags"]
```
Alternatively, this might be what you wanted:
```suggestion
    tag_list = ansible_dict_to_boto3_tag_list(tags)
    return [dict(ResourceType="network-interface", Tags=tag_list)]
```